### PR TITLE
add tests for pageCoverage

### PR DESCRIPTION
**Pull request by chaim-aharonson-applitools**
vg and classic should be unite

### CODE REVIEW

**Review by amitzur — APPROVED** [review]
:100:

File: java/overrides.js [hunk 1]
```diff
@@ -50,6 +53,10 @@
     'appium android check window': {skip: true}, //wrong ignore region ticket 2396
     'appium android check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
     'appium iOS check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
+	
+	'check window fully on page with horizontal scroll with css stitching': {skip: true}, //Could not start a new session
```
Comment (Yarden-Ingber) [inline java/overrides.js:57]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with css stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js [hunk 2]
```diff
@@ -50,6 +53,10 @@
...
+	'check window fully on page with horizontal scroll with scroll stitching': {skip: true}, //Could not start a new session
+	'appium android check region': {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```
Comment (Yarden-Ingber) [inline java/overrides.js:59]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"appium android check region": {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```

File: java/overrides.js [hunk 3]
```diff
@@ -38,6 +38,9 @@
     'check regions by coordinates in overflowed frame with css stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with scroll stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with vg': {skip: true}, // Unable to locate element: {"method":"css selector","selector":"#modal3 iframe"}
+    'check scrollable modal region by selector fully with css stitching': {config: {branchName: 'current1'}}, // diffs if compare to common baseline
+    'check scrollable modal region by selector fully with scroll stitching': {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```
Comment (Yarden-Ingber) [inline java/overrides.js:42]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with scroll stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

File: java/overrides.js [hunk 4]
```diff
@@ -38,6 +38,9 @@
     'check regions by coordinates in overflowed frame with css stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with scroll stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with vg': {skip: true}, // Unable to locate element: {"method":"css selector","selector":"#modal3 iframe"}
+    'check scrollable modal region by selector fully with css stitching': {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```
Comment (Yarden-Ingber) [inline java/overrides.js:41]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with css stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

File: java/overrides.js [hunk 5]
```diff
@@ -50,6 +53,10 @@
     'appium android check window': {skip: true}, //wrong ignore region ticket 2396
     'appium android check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
     'appium iOS check region with ignore region': {skip: true}, //wrong ignore region region ticket 2396
+	
+	'check window fully on page with horizontal scroll with css stitching': {skip: true}, //Could not start a new session
+	'check window fully on page with horizontal scroll with scroll stitching': {skip: true}, //Could not start a new session
```
Comment (Yarden-Ingber) [inline java/overrides.js:58]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with scroll stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js (hunk 1 above)
Comment (Yarden-Ingber) [inline java/overrides.js:57]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with css stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js (hunk 2 above)
Comment (Yarden-Ingber) [inline java/overrides.js:59]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"appium android check region": {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```

File: java/overrides.js (hunk 3 above)
Comment (Yarden-Ingber) [inline java/overrides.js:42]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with scroll stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

File: java/overrides.js (hunk 4 above)
Comment (Yarden-Ingber) [inline java/overrides.js:41]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with css stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

File: java/overrides.js (hunk 5 above)
Comment (Yarden-Ingber) [inline java/overrides.js:58]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with scroll stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js (hunk 1 above)
Comment (Yarden-Ingber) [inline java/overrides.js:57]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with css stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js [hunk 6]
```diff
@@ -38,6 +38,9 @@
     'check regions by coordinates in overflowed frame with css stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with scroll stitching': {skip: true}, // com.applitools.eyes.OutOfBoundsException: Region [(0, 15000) 385x5000, SCREENSHOT_AS_IS] is out of screenshot bounds [(0, 0) 385x15000, SCREENSHOT_AS_IS]
     'check regions by coordinates in overflowed frame with vg': {skip: true}, // Unable to locate element: {"method":"css selector","selector":"#modal3 iframe"}
+    "check scrollable modal region by selector fully with css stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
+    'check scrollable modal region by selector fully with scroll stitching': {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```
Comment (Yarden-Ingber) [inline java/overrides.js:42]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with scroll stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

File: java/overrides.js (hunk 2 above)
Comment (Yarden-Ingber) [inline java/overrides.js:59]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"appium android check region": {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```

File: java/overrides.js (hunk 5 above)
Comment (Yarden-Ingber) [inline java/overrides.js:58]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with scroll stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js (hunk 2 above)
Comment (Yarden-Ingber) [inline java/overrides.js:59]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"appium android check region": {skip: true}, //InvalidSelectorException: Locator Strategy 'class' is not supported for this session
```

File: java/overrides.js (hunk 5 above)
Comment (Yarden-Ingber) [inline java/overrides.js:58]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with scroll stitching": {skip: true}, //Could not start a new session
```

File: java/overrides.js (hunk 6 above)
Comment (Yarden-Ingber) [inline java/overrides.js:42]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "check scrollable modal region by selector fully with scroll stitching": {config: {branchName: 'current1'}}, // diffs if compare to common baseline
```

File: DotNet/overrides.js [hunk 7]
```diff
@@ -65,4 +65,6 @@
 	'adopted styleSheets on firefox': {skipEmit: true},
 	'check region by selector within shadow dom with vg': {skipEmit: true},
 	'check region by element within shadow dom with vg': {skipEmit: true},
+	'pageCoverage data is correct': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline DotNet/overrides.js:68]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"pageCoverage data is correct": {skipEmit: true},
```

File: java/overrides.js (hunk 1 above)
Comment (Yarden-Ingber) [inline java/overrides.js:57]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"check window fully on page with horizontal scroll with css stitching": {skip: true}, //Could not start a new session
```

File: DotNet/overrides.js [hunk 8]
```diff
@@ -65,4 +65,6 @@
...
+    'pageCoverage data is correct with vg': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline DotNet/overrides.js:69]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "pageCoverage data is correct with vg": {skipEmit: true},
```

File: java/mapping/supported.js [hunk 9]
```diff
@@ -15,7 +15,8 @@
     'visualGridOptions',
     'variationGroupId',
     'hooks',
-    'timeout']
+    'timeout',
+	'pageId']
```
Comment (Yarden-Ingber) [inline java/mapping/supported.js:19]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"pageId"]
```

Comment (Yarden-Ingber) [inline java/mapping/supported.js:19]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"pageId"]
```

File: DotNet/overrides.js (hunk 8 above)
Comment (Yarden-Ingber) [inline DotNet/overrides.js:69]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
    "pageCoverage data is correct with vg": {skipEmit: true},
```

File: DotNet/overrides.js (hunk 7 above)
Comment (Yarden-Ingber) [inline DotNet/overrides.js:68]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8914921)
```suggestion
	"pageCoverage data is correct": {skipEmit: true},
```

**Pull request merged**